runtest.sh

```diff
@@ -21,10 +21,12 @@ function print_style_fail_msg() {
     echo "Please run ${green}./runtest.sh${noColor} to check errors and fix them." 
 } 
 
-grep -V
-echo "set to +e"
+# grep -V
+# echo "set to +e"
```

Comment:
I think we can clean these up later.

Reply:
Agree.  This script needs some cleanup after we have a stable CI pipeline.